packages/react-vapor/src/components/table-hoc/reducers/TableRowReducers.ts

```diff
@@ -82,11 +87,12 @@ const TableRowActionReducers: {[key: string]: (...args: any[]) => any} = {
     [TableHOCRowActionsType.select]: selectTableRowReducer,
     [TableHOCRowActionsType.deselectAll]: deselectTableRowReducer,
     [TableHOCRowActionsType.toggleCollapsible]: toggleCollasibleTableRowReducer,
+    [TableHOCRowActionsType.closeAllToggleCollapsible]: closeAllCollapsibleRowReducer,
     [PerPageActions.change]: deselectTableRowReducer,
     [PaginationActions.changePage]: deselectTableRowReducer,
 };
 
-type ITableRowPayload = BasePayload | ITableRowAddPayload | ITableRowSelectPayload;
+type ITableRowPayload = BasePayload | ITableRowAddPayload | ITableRowSelectPayload | unknown;

```

Comment:
I am unsure what you want to do with that type modification

Reply:
My action is empty because I work on the whole table and not on a particular ID

```
const closeAllRowsReducer = (
    _state: HOCTableRowState[],
    _action: IReduxAction<unknown>
): ITableRowToggleCollapsiblePayload[] => [];
```